modules/database/src/ioc/dbStatic/link.h

```diff
@@ -77,8 +77,8 @@ struct macro_link {
     char *macroStr;
 };
 
-struct dbCommon;
-typedef long (*LINKCVT)();
+struct dbAddr;
+typedef long (*LINKCVT)(const void *, void *, const struct dbAddr *);
```

Comment:
Is this one of the other typedef's that this PR already changes?

Reply:
Yes, `LINKCVT` is the same as `FASTCONVERTFUNC` from dbConvertFast.h shown in the previous point. The definition of that has to be moved here for the result to build. This change is now incorporated into 2nd commit after a rebase+squash.